pkg/framework/owner.go

```diff
@@ -52,11 +52,12 @@ func AddOwnerReference(owner resource.KubernetesResource, scheme *runtime.Scheme
 		if !IsOwner(res, owner) {
 			owners := res.GetOwnerReferences()
 			falseValue := false
+			trueValue := true
 			owners = append(owners, v1.OwnerReference{
 				APIVersion:         gvk.GroupVersion().String(),
 				Kind:               gvk.Kind,
 				Name:               owner.GetName(),
-				Controller:         &falseValue,
+				Controller:         &trueValue,
```

Comment:
please create a new function or add a parameter for callers to control it, but this will add a regression to the BuildConfig and ImageStream creation. You can't have more than one controller owner in a given resource, IIRC.

Reply:
This change is no more required. We have removed it